Assets/Scripts/Game/Guilds/DarkBrotherhood.cs

```diff
@@ -79,6 +80,16 @@ public class DarkBrotherhood : Guild
 
         #region Guild Membership and Faction
 
+        public DarkBrotherhood()
+        {
+            RegisterEvents();
+        }
+
+        ~DarkBrotherhood()
+        {
+            UnregisterEvents();
```

Comment:
Not needed because when the destructor is called, it means events are already unregistered.

This is the case because the events themselves keep a strong reference on the instance so as long as they are registered, the instance can't be garbage collected.